Agents/FenlandTrajectoryAgent/docker-compose-debug.yml

```diff
@@ -0,0 +1,11 @@
+version: '3.8'
+services:
+  fenland_trajectory_agent:
```

Comment:
I would strongly recommend to stick to the convention of using hyphens: `fenland-trajectory-agent`. That way this agent will be consistent with most other agents. Please note that this applies to all four docker compose files.